### DISCUSSION TITLE
Extend bins over side padding

### DISCUSSION THREAD

**Discussion by arielzusya** [Le0Michine/FusionGridfinityGenerator#139] [Ideas]
Love your add-on. Makes Gridfinity so much easier (which is saying a lot because it is pretty easy to start with)!

When the grid doesn't fit perfectly in a drawer I use your add side padding feature to get a perfect fit. I'd love to see a similar option in the bin creator. The way I imagine it, the base would remain the same up to the edge of the grid, and then the underside of the bin would be flat above the side padding while the bin itself would seamlessly extend over the padding. That would be so helpful!

Attached is a quick render of a recent stl I did where I strongarmed the box in this way.
![simple 3x2 with overhang](https://github.com/user-attachments/assets/ad232169-0397-4e3f-b5a1-6868816729c5)

**Comment by Le0Michine** [comment]
this would require supports when printing though, are you working around that somehow?

**Comment by arielzusya** [comment]
I usually do minimal tree supports when I have that overhang unless the bin isn't hollow and then often I can print it upright on a side with no supports. Seems to work, though admittedly I am relatively new to printing and modeling for printing (came from a woodworking background and only recently got a printer). If there is an easier way, I'd love to learn it. I thought about extending that side of the model down to match the height of the base and then not have the padding on the grid, but generally I've found it easier to just use supports on the bins with overhangs.

**Reply by mattshack1** [reply]
I am trying to replicate expanding a bin to account for padding but can't figue it out. I used the bin generator to make a bin and am trying to now expand a side cover the padding. Can you explain how you did it?
